templates/security/livepatch.html

```diff
@@ -68,9 +56,45 @@ <h2>Part of Ubuntu Advantage</h2>
   </div>
 </section>
 
-{% with colour="light", bordered="true" %}{% include "shared/_call-sales.html" %}{% endwith %}
+<section class="p-strip--suru-accent is-dark">
+    <div class="row">
+      <div class="col-3 p-card u-hide--small u-vertically-center">
+        {{ image (
+            url="https://assets.ubuntu.com/v1/c830d547-GMO-Pepabo-logo.svg",
+            alt="",
+            width="572",
+            height="132",
+            hi_def=True,
+            loading="auto"
+            ) | safe
+          }}
+      </div>
+      <div class="col-8 col-start-large-5">
+        <div class="p-heading-icon--small">
+          <div class="p-heading-icon__header">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/fc781c43-Case+study+-+white.svg",
+              alt="",
+              width="32",
+              height="32",
+              hi_def=True,
+              attrs={"class":"p-heading-icon__img"},
+              loading="lazy"
+              ) | safe
+            }}
+            <p class="p-muted-heading u-sv3" style="color: #fff; line-height: 1.5rem;">Case study</p>
+          </div>
+          <h2 class="p-heading--4">GMO Pepabo cuts OPEX costs with live kernel patching</h2>
+          <p>As a leading provider of consumer-facing internet services, GMO Pepabo cannot afford downtime to its mission-critical workloads. To minimise OPEX costs, it needed a way to apply kernel updates without rebooting servers.</p>
+          <p><a href="https://pages.ubuntu.com/rs/066-EOV-335/images/GMO%20Pepabo%20Case%20study%208.pdf" class="p-button--neutral">Read more in the case study</a></p>
+        </div>
```

Comment:
this should be an engage page link? https://ubuntu.com/engage/case-study-gmo-pepabo